custom_components/blitzortung/sensor.py

```diff
@@ -1,14 +1,13 @@
 import logging
 
-from homeassistant.const import ATTR_ATTRIBUTION, CONF_NAME, LENGTH_KILOMETERS
-from homeassistant.components.sensor import SensorEntity
+from homeassistant.const import ATTR_ATTRIBUTION, CONF_NAME, DEGREE, UnitOfLength
+from homeassistant.components.sensor import SensorDeviceClass, SensorEntity
 from homeassistant.helpers.device_registry import DeviceEntryType
 
 from .const import (
     ATTR_LAT,
     ATTR_LIGHTNING_AZIMUTH,
     ATTR_LIGHTNING_COUNTER,
-    ATTR_LIGHTNING_DISTANCE,
```

Comment:
Let's also replace `ATTR_LIGHTNING_DISTANCE` with `SensorDeviceClass.DISTANCE` in `__init__.py` and remove `const.ATTR_LIGHTNING_DISTANCE` at all.

Reply:
done